frontend/src/admin/createUser/CreateUserPageContents.jsx

```diff
@@ -0,0 +1,67 @@
+import React from "react";
+import PropTypes from "prop-types";
+
+import FormConsole from "../../general/userForm/FormConsole";
+import FormFailedDialog from "../../general/userForm/FormFailedDialog";
+import { createUser } from "../../api/UserManagementApi";
+
+export default class CreateUserPageContents extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      dialogMessage: ""
+    };
+    this.dialogElement = React.createRef();
+    this.handleSubmit = this.handleSubmit.bind(this);
+    this.openDialog = this.openDialog.bind(this);
+    this.setDialogMessage = this.setDialogMessage.bind(this);
+  }
+
+  handleSubmit(username, password) {
+    const { history } = this.props;
+    createUser({ username, password })
+      .then(() => {
+        history.push("/Home");
+      })
+      .catch((error) => {
+        this.openDialog();
+        this.setDialogMessage(error.message);
+      });
+  }
+
+  setDialogMessage(message) {
+    this.setState({
+      dialogMessage: message
+    });
+  }
+
+  openDialog() {
+    this.dialogElement.current.openDialog();
+  }
+
+  render() {
+    const { dialogMessage } = this.state;
+    return (
+      <>
+        <FormConsole
+          handleSubmit={this.handleSubmit}
+          buttonName="Create"
+          isValidate
+          isCreateUser
+        />
+        <FormFailedDialog
+          ref={this.dialogElement}
+          title="Failed to create user"
```

Comment:
`title` can be hard coded in the `<FormFailedDialog/>` component

Reply:
Good catch!

Reply:
Oh wait, I know why I didn't hardcode the title, it's because I'm going to use FormFailedDialog for also Login and not just Create User, which is why I had it as a prop.